src/core/object/mod.rs

```diff
@@ -13,7 +13,7 @@ pub struct UntypedObject(pub(crate) Map<String, Json>);
 // TODO: Replace anyhow error type.
 /// A strongly typed parameter that can represent metadata entries or request parameters.
 pub trait TypedParameter:
-    TryFrom<Json, Error = anyhow::Error> + Into<Json> + Clone + std::fmt::Debug
+    TryFrom<Json, Error = anyhow::Error> + TryInto<Json> + Clone + std::fmt::Debug
```

Comment:
Add TryInto::Error type restriction to use anyhow::Error.

Reply:
Adding the error type will cause conflicts with the default From<T> infallible error type. Leaving off the TryInto::Error type.